### DISCUSSION TITLE
blog/notitle

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#2] [Announcements]
# blog/notitle

中国人为什么还要穷尽一生的时间，做英美人这些智力游戏的应声虫？

https://uselessulysses.com/blog/notitle

**Comment by Pathsis** [comment]
☺️

**Reply by Young-Mann** [reply]
博主最近换博客主题换得好勤快，是因为搜集了不少好看的主题嘛😂

**Reply by Pathsis** [reply]
别提了，堕落了快俩月了，本来还是想继续jekyll，毕竟现在这个框架一堆JS，根本不会弄，但不换觉的自己之前的折腾更蠢了😄

**Reply by Young-Mann** [reply]
哈哈，也许可以考虑浅学一下前段三件套？这样魔改博客也更得心应手～

**Reply by Pathsis** [reply]
前端三件套文盲。本来想着可以省点时间，但有这俩月学习，早就今非昔比了，现在连const和export是啥都看不懂😄

**Reply by Young-Mann** [reply]
我觉得，如果想更自由地魔改博客主题，那前端三件套的使用方式还是有必要了解的。不过，倒也不必练到能够找前端岗工作的水平，仅需要对三件套各自的基础概念有所了解，能够明白：1. 这 3 个工具分别能干什么；2. 假如我碰到没见过的语法、标签或者关键词，我是不是知道自己大概能在哪里找到相关资料，并且能用它们做出很简单的 demo ，就可以放宽心开始尝试魔改了。

如果要练手三件套，从魔改静态博客的主题开始或许是一个选择。在这一过程中， html 和 css 会被反复用到，因此会越来越熟练；至于 js 能练到多少，就看自己在魔改的过程中是否想添加新功能、又想添加多少新功能了。

写到这里，感觉自己太啰嗦，还有点好为人师，希望不会冒犯到博主（如果有冒犯到，那就当我发过没有这条回复吧😂）。

**Reply by Pathsis** [reply]
多谢建议！感觉确实到达一个瓶颈了，不自己学习而只求修改别人的，恐怕会更耗费时间，等忙完手中的活，准备系统学习下。另外，你很有心得啊，看来以前没少折腾哈哈
